ui/src/app/services/patch-db/patch-db.service.ts

```diff
@@ -39,66 +41,95 @@ export class PatchDbService {
   ) { }
 
   async init (): Promise<void> {
-    await this.initSource(this.wsSource)
-    const wsSuccess = !!(await this.storage.get(this.WS_SUCCESS))
-    this.wsSuccess$.next(wsSuccess)
+    const cache = await this.bootstrapper.init()
+    this.sources$.next([this.wsSource, this.http])
+
+    this.patchDb = new PatchDB(this.sources$, this.http, cache)
+
+    this.patchConnection$.next(PatchConnection.Initializing)
+    this.data = this.patchDb.store.cache.data
   }
 
-  start (): void {
-    const a = this.patchSub ? 'RESTARTING ' : 'STARTING '
-    const b = !!this.polling$.getValue() ? 'POLL' : 'WEBSOCKET'
-    console.log('patchDB: ' + a + b)
-
-    this.patchSub = this.patchDb.sync$()
-    .pipe(
-      debounceTime(420),
-      withLatestFrom(this.patchConnection$, this.wsSuccess$, this.polling$),
-      tap(async ([cache, connection, wsSuccess, polling]) => {
-        this.bootstrapper.update(cache)
-
-        if (connection === PatchConnection.Initializing) {
-          console.log(polling ? 'patchDB: POLL CONNECTED' : 'patchDB: WEBSOCKET CONNECTED')
-          this.patchConnection$.next(PatchConnection.Connected)
-          if (!wsSuccess && !polling) {
-            console.log('patchDB: WEBSOCKET SUCCESS')
-            this.storage.set(this.WS_SUCCESS, 'true')
-            this.wsSuccess$.next(true)
+  async start (): Promise<void> {
+    await this.init()
+
+    console.log('PATCH DB', this.patchDb)
+
+    this.subs.push(
+
+      // Connection Error
```

Comment:
I asked that the signals get split into individual to separate concerns + understand